Terraform/AWS/task3/VPCinAWS.tf

```diff
@@ -43,24 +34,19 @@ resource "aws_internet_gateway" "igw" {
     Name = "${var.env_code}-IGW"
   }
 }
-resource "aws_nat_gateway" "natgw1" {
-  subnet_id     = aws_subnet.public1.id
-  allocation_id = aws_eip.eip1.id
+resource "aws_nat_gateway" "natgw" {
+  count = 2
```

Comment:
could you change the `count = 2` to `count = length(local.public_cidr)` as it returns the value 2